package.json

```diff
@@ -15,7 +15,8 @@
     "test": "run-s eslint test:unit",
     "test:unit": "mocha",
     "prepublish": "npm prune && run-s build",
-    "watch": "npm run compile -- --watch"
+    "watch": "npm run compile -- --watch",
+    "report": "node_modules/allure-commandline/bin/allure generate .allure-results --clean && node node_modules/allure-commandline/bin/allure open"
```

Comment:
I would say this would work:
```json
"report": "run-s report:*",
"report:generate": "allure generate .allure-results --clean",
"report:open": "allure open"
```

Can you try this? It looks quite simpler.

Reply:
Sure. But without path to allure it didn't work.

Reply:
I am sure it is windows issue...

Reply:
I am working on Mac OS ;)

Reply:
don't know. Perfectly works without path for me.

Reply:
@netmajor How exactly it does not work? What error do you see?
In general, this is NPM behavior, when it picks up executable files from `node_modules`. it works on Windows, Mac and Linux, AFAIK

Reply:
```
Error: Cannot find module '/Users/Netmajor/Documents/wdio-allure-reporter/allure'
```